src/adapters/http/uploadToS3/router.ts

```diff
@@ -21,6 +21,7 @@ const handler =
       E.ap(t.union([t.undefined, AmzSdkChecksumAlg]).decode(req.headers['x-amz-sdk-checksum-algorithm'])),
       E.ap(AmzMetaSecret.decode(req.headers['x-amz-meta-secret'])),
       E.ap(AmzChecksumSHA256.decode(req.headers['x-amz-checksum-sha256'])),
+      E.ap(E.of(req.body)),
```

Comment:
Can the property `body` be `undefined`? Do you think we should include validation?

Reply:
We can add the validation in a future PR